### ISSUE TITLE
hide tray icon

### ISSUE THREAD

**Issue by fchud** [nefares/Background-Muter#22]
hi again!

would love to see an option to hide tray icon.

also,
launching second process while there is an active one, should just open the gui of the first one.
and this should solve the problem of activating a hidden gui without icon.

**Comment by nefares** [comment]
1) hide tray icon => Do you mean when it's shown (not minimized to tray)?
2) I agree regarding the single instance issue. Will mark as a bug

**Comment by fchud** [comment]
> 1. Do you mean when it's shown (not minimized to tray)?

no, i want to be able to completely hide the icon, so it does not clutter tray area while still working in background.

like _**4t Tray Minimizer**_ or _**TranslucentTB**_ or _**WindowSpace**_ do.

**Comment by Valour549** [comment]
I want to change the tray icon to something not a cat. Lol it doesn't really fit the purpose of this program~

I tried replacing the icon group using Resource Hacker, but for some reason the tray icon is still a cat.

Please advise, thank you.

**Comment by nefares** [comment]
Both parts are relevant:
1. **Hide tray icon** — feature not yet implemented.
2. **Second instance should open existing GUI** — this is being implemented as part of #43 (single-instance mutex). Once a mutex is in place, the second launch can detect the existing instance.
